src/main/java/src/main/java/EventModel.java

```diff
@@ -0,0 +1,36 @@
+package src.main.java;
+
+public class EventModel {
+    private String nameEvent;
+    private String body;
+
+    public EventModel(){
+
+    }
+
+    public EventModel(String nameEvent,String body){
+        this.nameEvent=nameEvent;
+        this.body=body;
+    }
```

Comment:
```suggestion
     public EventModel(String nameEvent, String body) {
        this.nameEvent = nameEvent;
        this.body = body;
    }
```
lisibilité et propreté du code ;). A appliquer sur le reste de la classe please